packages/at_onboarding_cli/lib/src/onboard/at_onboarding_service_impl.dart

```diff
@@ -174,12 +180,13 @@ class AtOnboardingServiceImpl implements AtOnboardingService {
     return enrollmentResponse;
   }
 
-  void _listenToPkamSuccessStream(
+  Future<void> _listenToPkamSuccessStream (
```

Comment:
Made _listenToPkamSuccessStream return a Future which is only completed when we've actually processed the success